libs/gi-formula/src/executors/gen-desc/executor.ts

```diff
@@ -18,14 +18,16 @@ export default async function runExecutor(
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const { entries } = require('../../data')
   const { outputPath } = options
-  const { conditionals } = metaList
   const formulas: Record<string, Record<string, any>> = {}
 
   // Massage data from `metaList`
-  for (const [src, entries] of Object.entries(conditionals!)) {
-    for (const [name, obj] of Object.entries(entries!)) {
-      entries![name] = { src, name, tag: { src, name }, ...obj }
-    }
+  const conditionals: Record<string, Record<string, any>> = {}
+  for (const { tag, meta } of metaList.conditionals!) {
+    const { src, q } = tag // Conditionals guarantee `src-q` pair uniqueness
+    if (!conditionals[src!]) conditionals[src!] = {}
+    if (!conditionals[src!][q!])
+      conditionals[src!][q!] = { src, name: q, tag, ...meta }
```

Comment:
```suggestion
      conditionals[src!][q!] = { src, q, tag, ...meta }
```
This allows easier usage of the tag later when fetching from the calc

Reply:
nvm